src/global/actions/ui/messages.ts

```diff
@@ -430,7 +430,10 @@ addActionHandler('focusMessage', (global, actions, payload): ActionReturnType =>
 
   if (replyMessageId) {
     const replyStack = selectReplyStack(global, chatId, threadId, tabId) || [];
-    global = replaceTabThreadParam(global, chatId, threadId, 'replyStack', [...replyStack, replyMessageId], tabId);
+
+    if (replyMessageId !== replyStack[replyStack.length - 1]) {
```

Comment:
What happens if `replyStack` is an empty array?

Reply:
In this case, the condition will not be satisfied because `replyStack[-1]` equals `undefined`, and this does not cause any errors to be thrown.

Reply:
Clicking between two replies adds them to the reply stack
![image](https://github.com/Ajaxy/telegram-tt/assets/19638254/8deaec1d-95f7-4844-8f26-f7bf45fc0ebc)